arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -4676,6 +4631,14 @@
 			path = "Display utility associations";
 			sourceTree = "<group>";
 		};
+		F1D7F6D0278FB595004968E3 /* Shapefiles */ = {
```

Comment:
Because the portal item: https://arcgis.com/home/item.html?id=15a7cbd3af1e47cfa5d2c6b93dc44fc2 downloads as a zip archive, the folder reference needs to go under Portal Data/Archives